sourcemap/elasticsearch.go

```diff
@@ -119,12 +116,13 @@ func (s *esStore) runSearchQuery(ctx context.Context, name, version, path string
 
 func parse(body io.ReadCloser, name, version, path string, logger *logp.Logger) (string, error) {
 	var esSourcemapResponse esSourcemapResponse
-	if err := json.NewDecoder(body).Decode(&esSourcemapResponse); err != nil {
+	var buf bytes.Buffer
+	if err := json.NewDecoder(io.TeeReader(body, &buf)).Decode(&esSourcemapResponse); err != nil {
```

Comment:
sorry if it's obvious, but I don't understand why we're using an `io.TeeReader` here, could you please explain? Is it because some benefit unknown to me or just a debugging leftover?

Reply:
yep, this :see_no_evil: